src/test/java/Hmwk21.java

```diff
@@ -1,2 +1,74 @@
-public class Hmwk21 {
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+
+//Using Action Class methods, create/refactor the test case renamePlaylist()
+//        Implement Explicit Waits
+//        Remove all Thread.sleep implementations
+//        Create a new branch and commit your changes
+//        Push your code to a remote repository
+//        Create a pull request
+//        Copy and paste the link of the pull request (or your branch) to the field below, so we can check your homework
+public class Hmwk21 extends BaseTest{
```

Comment:
**Naming Conventions in Java:**  Class names should be nouns

For example 

`NewNamePlayListTest`

![image](https://user-images.githubusercontent.com/104761482/200247549-2063342f-326d-45c3-b8f2-ac496514ee53.png)